ast/vcl_types.go

```diff
@@ -14,6 +26,14 @@ func (i *Ident) GetMeta() *Meta { return i.Meta }
 func (i *Ident) String() string {
 	return i.LeadingInlineComment() + i.Value + i.TrailingComment()
 }
+func (i *Ident) IsValueType() bool {
+	for _, v := range valueTypes {
+		if v == i.Value {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
![Suggestion](https://img.shields.io/badge/review-suggestion-red)
Ident will express kind of types so this function should be a helper function in `interpreter` packages somewhere instead of `ast` method.

Reply:
Realized there's already a set of valid value types defined in `linter/helper.go`, should have remembered that since I removed `ID` from it. 😅 Makes sense to me to reuse that same map for both the linter and the interpreter. I've moved that map to the types package and used that instead of the IsValueType method. Thoughts?